src/components/chart/examples/chart-resizable-container.scss

```diff
@@ -0,0 +1,64 @@
+* {
+    box-sizing: border-box;
+}
+
+:host {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+
+    resize: both;
+    overflow: auto;
+
+    box-sizing: border-box;
+    position: relative;
```

Comment:
Sonarcloud is complaining about this since it's already declared on line 6 :slightly_smiling_face: